lib/src/layer/polygon_layer.dart

```diff
@@ -26,6 +26,11 @@ class PolygonLayerOptions extends LayerOptions {
   }
 }
 
+enum PolygonLabelPlacement {
+  centroid,
+  polylabel,
+}
```

Comment:
I also think we might need to rename these options to make them clearer to other people. Perhaps:

```suggestion
enum PolygonLabelPlacement {
  squareCenter,
  polyCenter,
}
```

Reply:
Ok, @ibrierley made a good point on this one, so I won't change it...